authors/apps/authentication/serializers.py

```diff
@@ -22,7 +22,7 @@ class Meta:
         model = User
         # List all of the fields that could possibly be included in a request
         # or response, including fields specified explicitly above.
-        fields = ['email', 'username', 'password','token','complete_profile']
```

Comment:
hey @nakimera thanks for the great work i think this line was meant to be there don't you think

Reply:
hey @j0flintking02. Thanks for the feedback. The line is actually still there. I just modified it by removing the complete profile field. A user can only use the app after activating their account. This includes completing their profile.  They get the prompt to complete their profile after activating the account.

Reply:
hey @JoyLubega. Thanks for pointing out that the user doesnot need to login. I had missed that. I however think it is necessary to prompt them that their accout has been activated  and they need to complete their profile.